tensorflow_addons/layers/embedding_bag.py

```diff
@@ -49,8 +49,13 @@ def _embedding_bag(
     Returns:
       A `Tensor` of the format specified by `data_format`.
     """
-    if weights is None:
+    if weights is None and combiner == "sum":
```

Comment:
Use reduction instead of combiner

Reply:
The combiner is combining part of the layer and applies to the inputs/weights, not the output/loss! Is reduction still the right approach there?

Reply:
Could you add ```combiner``` document?